apps/api/src/auth/auth.controller.ts

```diff
@@ -48,6 +49,18 @@ export class AuthController {
     }
   }
 
+  @Post('/google/idtoken')
+  async authWithIdToken(
+    @Body() body: { idToken: string },
+    @Res({ passthrough: true }) res: Response
+  ) {
+    const { idToken } = body
```

Comment:
Has the `body` been validated before destructuring? Could you try sending some malformed request and ensure that it would 400?

Reply:
Yes, it returns 400 for malformed requests due to `undefined` is not a valid token. Anyway, it will handle that explicitly.